### PR TITLE
added link to signature spec from troubleshooting entry

### PR DESCRIPTION
**Pull request by zr-msft**

### CODE REVIEW

File: content/en/docs/troubleshooting.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ The output contains all the certificate information used for signing the artifac
 
 The output also contains details about the artifact that was signed, such as the digest. You can use this information to confirm that the correct artifact was signed.
 
+For more information about signatures and their components, see [the Notary signature specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md).
```
Comment (yizha1) [inline content/en/docs/troubleshooting.md:76]: Adding this reference is good. But the specification link may be changed later.

**Review by yizha1 — COMMENTED** [review]
Thanks @zr-msft

**Review by FeynmanZhou — APPROVED** [review]
LGTM

File: content/en/docs/troubleshooting.md (hunk 1 above)
Comment (yizha1) [reply content/en/docs/troubleshooting.md:76]: Based on the latest discussion, the repository will be renamed from `notaryproject` to `specifications`. See [Rename notaryproject/notaryproject repository to notaryproject/specifications](https://github.com/notaryproject/.github/issues/38).

**Comment by zr-msft** [comment]
@FeynmanZhou  and @yizha1 rebased to resolve conflict. Can you approve again so I can merge?

File: content/en/docs/user-guides/troubleshooting.md [hunk 2]
```diff
@@ -73,6 +73,7 @@ The output contains all the certificate information used for signing the artifac
 
 The output also contains details about the artifact that was signed, such as the digest. You can use this information to confirm that the correct artifact was signed.
 
+For more information about signatures and their components, see [the Notary signature specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md).
```
Comment (yizha1) [inline content/en/docs/user-guides/troubleshooting.md:76]: Thanks @zr-msft. Maybe we should use the link to `v1.0.0` spec?

Comment (zr-msft) [reply content/en/docs/user-guides/troubleshooting.md]: thank you @yizha1 

fixed

**Comment by zr-msft** [comment]
@FeynmanZhou mind talking a look so i can merge?

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**